units/INU2003/INU2003_Script.lua

```diff
@@ -1,17 +1,18 @@
 -- T2 missile launcher
 
+local AddBombardModeToUnit = import('/lua/nomadsutils.lua').AddBombardModeToUnit

```

Comment:
becaue anchor go down and is replaced by bombardment. So when it only remove anchor then will need make another comit that also add bombardment.  And one comit with other will not have balanced unit.